src/js/components/Share/ShareButtonDesktopTablet.jsx

```diff
@@ -52,6 +54,42 @@ class ShareButtonDesktopTablet extends Component {
     const googleCivicElectionId = 0;
     let kindOfShare = 'BALLOT';
     let whatAndHowMuchToShare;
+
+    // Add debug logging
+    console.log('Share button clicked');
+    console.log('Current pathname:', window.location.pathname);
+    console.log('VoterWeVoteId:', VoterStore.getVoterWeVoteId());
+
+    // Add dataLayer tracking
+    const { location: { pathname: currentPathname } } = window;
+    const page = lookupPageNameAndPageTypeDict(currentPathname);
+    
+    const dataLayerObject = {
+      event: 'shareClick',
```

Comment:
event should be 'action'. Please also add actionDetails, which would include the id of the button clicked ("shareButtonDesktopTablet") -- BUT I think we should adjust this id to include the kind of share (ex/ candidateShare, officeShare, etc.):
```
      actionDetails: {
        actionType: 'openModal',
        buttonId: <ADD BUTTON ID HERE>,
      },
```